src/KeyVault/KeyVault/help/New-AzKeyVaultCertificatePolicy.md

```diff
@@ -71,6 +71,13 @@ Updated                         :
 
 This command creates a certificate policy that is valid for six months and reuses the key to renew the certificate.
 
+### Example 2
+Creates an in-memory certificate policy object. (autogenerated)
+
+```powershell <!-- Aladdin Generated Example --> 
+New-AzKeyVaultCertificatePolicy -IssuerName 'Self' -KeyType RSA -RenewAtNumberOfDaysBeforeExpiry <System.Int32> -SecretContentType application/x-pkcs12 -SubjectName 'CN=contoso.com' -ValidityInMonths 6
```

Comment:
We only want the portion of the datatype after the last dot to be displayed. <System.Int32> should be
<Int32> in this example.

This is because the full datatype name can be very long for some parameters such as <Microsoft.Azure.Commands.DataFactories.Models.PSDataFactory>.

Reply:
Sure, can you elaborate on how do we want this type to look? System.Collections.Generic.List1[System.String], would it be List1[String]?

Reply:
It would be `<string>`